crates/hir-def/Cargo.toml

```
@@ -1,7 +1,8 @@
 [package]
 name = "hir-def"
 version = "0.0.0"
-description = "TBD"
+repository = "https://github.com/rust-lang/rust-analyzer"
+description = "Everything between macro expansion and type inference for rust-analyzer."
```

```suggestion
description = "Early name and import resolution for rust-analyzer."
```
might be more fitting